include/RAJA/index/ListSegment.hpp

```diff
@@ -70,7 +70,7 @@ class TypedListSegment
  * won't see any different usage or behavior.
  */
   
-#if ((defined(__NVCC__) || (defined(__clang__) && defined(__CUDA__))) && defined(RAJA_ENABLE_CUDA)) || defined(RAJA_ENABLE_HIP)
+#if defined(RAJA_CUDA_ACTIVE) || defined(RAJA_ENABLE_HIP)
```

Comment:
RAJA_ENABLE_DEVICE?

Reply:
Not for our use case here. By using RAJA_CUDA_ACTIVE this lets us compile with non-nvcc compilers when CUDA is enabled.

Reply:
All I was saying is that the definition of RAJA_ENABLE_DEVICE is defined(RAJA_CUDA_ACTIVE) || defined(RAJA_ENABLE_HIP)

Reply:
Ah you are right! sorry I confused it with RAJA_CUDA_ACTIVE.

Reply:
Ya I thinkthe use of DEVICE is a little confusing since we already have a RAJA_DEVICE_CODE macro

Reply:
Should this be RAJA_DEVICE_ACTIVE because it depends on RAJA_CUDA_ACTIVE instead of RAJA_ENABLE_CUDA?